datatypes/src/collections/batch_builder.rs

```diff
@@ -328,7 +328,7 @@ impl RawFeatureCollectionBuilder {
             .add_buffer(values_buffer);
 
         let data = if let Some(nulls) = nulls_buffer {
-            builder.null_bit_buffer(nulls).build()
+            builder.null_bit_buffer(Some(nulls)).build()
```

Comment:
the "nulls" buffer is already an option. Can't we use that? Maybe the if else block is obsolete now?

Reply:
good catch!
[a0faed2](https://github.com/geo-engine/geoengine/pull/556/commits/a0faed2feeee14341533dc5edd8282dc612319f0)